task.go

```diff
@@ -116,6 +116,8 @@ func (r *marathonClient) KillApplicationTasks(id string, opts *KillApplicationTa
 //	opts:		KillTaskOpts request payload
 func (r *marathonClient) KillTask(taskID string, opts *KillTaskOpts) (*Task, error) {
 	appName := taskID[0:strings.LastIndex(taskID, ".")]
+	appName = strings.Replace(appName, "_", "/", -1)
```

Comment:
Do we only need the replacement in this case? What about other calls to Marathon? I'm not saying that we need to fix everything along this PR, just that we should create a ticket if there's more sanitization to do.

Did you add/extend a test for the replacement?